bin/package

```diff
@@ -5863,7 +5863,7 @@ cat $j $k
 				echo "$command: $var changed from $old to $new" >&2
 				err=y ;;
 			esac
-		else	case $new in
+		else	[ -d $INSTALLROOT/lib/package/gen ] && case $new in
```

Comment:
This code consistently uses `test` instead of `[`, let's maintain that consistency.